### DISCUSSION TITLE
Not auto reloading after new deployment of nextjs app on vercel.

### DISCUSSION THREAD

**Discussion by mattscepter** [vercel/next.js#38981] [Help]
If a tab has been opened with the website and a new code has been deployed to the production. As soon as we click on any link or navigate to a new page in the tab previously opened 404 page is triggered. But reloading the page fixes the issue.

The browser is trying to fetch page json on the old buildID but the buildID has changed and hence i ma getting 404 error.
Making the buildID contant is causing the 500 error as the cache for static js chunks is not getting updated.

Stack being used- apollo, nextjs

Any suggestions would be helpful.

**Comment by javlund** [comment]
Hey @mattscepter - I seem to have the same or a similar issue: https://github.com/vercel/next.js/issues/39535

Did you find a solution for this?
